CHANGELOG.md

```diff
@@ -67,10 +67,12 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 - Improve runtime of some unit tests. #1125
 - Run curl OR wget (not both) when attempting to communicate as a new user on
   Linux. #1407
+- Renamed Communicate as new user to Communicate as backdoor user. #1410
```

Comment:
In the CHANGELOG, we track the PR number, not the issue number. :)

```suggestion
- Renamed CommunicateAsNewUser to CommunicateAsBackdoorUser. #1433
```

Reply:
@shreyamalviya Incorrect. Track the issue number when possible. The issue links to the PR and generally gives more context about why the change was made.

This is a decision made on a case-by-case basis, however. The goal is to communicate information about the change to the user.

Reply:
Oh, the impression I gathered from the CHANGELOG was that we always link the PR number. My bad!

Reply:
It's on a case-by-case basis. We should prefer the issue number. However, we should use whatever provides the clearest information/story to the reader.